internal/testprotos/desc_test_complex.proto

```diff
@@ -95,6 +100,10 @@ message Another {
     option (a).test.array = 2;
     option (a).test.(.foo.bar.Test.Nested._NestedNested._garblez) = "whoah!";
 
+	option (map_vals).vals = {}; // no key, no value
```

Comment:
This is the reproducer case, with a few other cases below just to be thorough...